test/e2e/framework/service/jig.go

```diff
@@ -54,6 +54,8 @@ import (
 // NodePortRange should match whatever the default/configured range is
 var NodePortRange = utilnet.PortRange{Base: 30000, Size: 2768}
 
+var errAllocated = errors.New("provided port is already allocated")
```

Comment:
we should have a comment over this one too, explaining where it was copied from.
this is mainly because we use it for string matching and the origin has to be clear.

Reply:
sure